mono/mini/mini-s390x.c

```diff
@@ -4677,10 +4677,11 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 		case OP_MEMORY_BARRIER:
 			s390_mem (code);
 			break;
-#if USE_COOP_GC
 		case OP_GC_SAFE_POINT: {
```

Comment:
As a matter of fact, that should never be reached, as in https://github.com/mono/mono/blob/master/mono/mini/mini-ops.h#L1080, this opcode is not defined for s390x.